Gemfile.lock

```diff
@@ -41,7 +41,6 @@ PATH
 
 GEM
   remote: https://rubygems.org/
-  remote: https://enterprise.contribsys.com/
```

Comment:
kill the Gemfile.lock changes since they'll kick sidekiq EE out of our build

Reply:
done